### PR TITLE
Minor string re-arranging, create desc for "wallpaper depth", and create section "wallpaper" with wallpaper-related settings

### PR DESCRIPTION
**Pull request by SuperDragonXD**
The title says it all.

### CODE REVIEW

**Comment by suphon-t** [comment]
Looks good overall, but I think app market sounds strange. Keeping store would be better imo.

**Comment by SuperDragonXD** [comment]
Okay

**Comment by SuperDragonXD** [comment]
Done!

File: lawnchair/res/values/strings.xml [hunk 1]
```diff
@@ -298,25 +303,32 @@
...
     <string name="backup_restore_error">Failed to restore backup.</string>
     <string name="invalid_backup_file">Invalid backup file.</string>
+
     <string name="google_folder_title" translatable="false">Google</string>
-    <string name="battery_charging_percentage_charging_time">"%1$d%% — Full in %2$s"</string>
-    <string name="wallpaper_depth_effect">Wallpaper Depth Effect</string>
-    <string name="gesture_double_tap">Double Tap</string>
+
+    <string name="wallpaper_depth_effect_label">Wallpaper Depth Effect</string>
+    <string name="wallpaper_depth_effect_description">Zoom in and out of the wallpaper when transitioning between states</string>
```
Comment (ghost) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="wallpaper_depth_effect_description">Zoom into and out of the wallpaper when transitioning between states.</string>
```

File: lawnchair/res/values/strings.xml [hunk 2]
```diff
@@ -298,25 +303,32 @@
...
+
+    <string name="gestures_label">Gestures</string>
+
+    <!-- Gestures -->
+    <string name="gesture_double_tap">Double-Tap</string>
```
Comment (ghost) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="gesture_double_tap">Double Tap</string>
```

**Review by ghost — CHANGES_REQUESTED** [review]
Hello! Thanks for your interest in contributing. I enclose two minor suggestions.

**Comment by SuperDragonXD** [comment]
Late reply, but also done.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Thanks, @SuperDragonXD!

**Pull request merged**